readthedocs/rtd_tests/tests/test_user_admin_actions.py

```diff
@@ -0,0 +1,34 @@
+from unittest.mock import patch
```

Comment:
We are creating new test files where they belong and avoiding this old pattern where all the files were all together. Can you move this file to `readthedocs/core/tests/` instead?

Reply:
Oh, sure!